### DISCUSSION TITLE
Empty body after visiting 404 page in laravel octane

### DISCUSSION THREAD

**Discussion by yilanboy** [livewire/livewire#8078] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.7

### Laravel version

v10.47.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Create a livewire full page component, and its route is `/users/{userId}`
2. Start the octane server, and run your livewire app on it
3. Visit the exists user page, for example: `/users/1`
4. Visit the non-exists user page, for example: `/users/999` (Assume that the user with id 999 does not exist in the database)
5. Visit the exists user page again, for example: `/users/2`, then you will get empty body, there is nothing in `<body>`

### Repository or Wirebox URL

https://wirebox.app/b/xp32l

### Additional information

This bug will only show up when your livewire app is **on a octane server**, the wirebox link is just a simple sample code.

After you visit the non-exists user page, the other exists user pages will return a empty body.

The Laravel octane version is v2.3.5, and swoole version is 5.1.1.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by gazzoy** [comment]
I see exact the same error on the same situation. Not sure why it happens, but I also belive its a Livewire's bug as I don't see this error without Livewire.

**Reply by gazzoy** [reply]
@YilanBoy 

As a workaround you can add ` --max-requests=1` to `ocane:start` command, but obviously it's not recommended for production :(

```
[program:octane]
command=/usr/bin/php -d variables_order=EGPCS /var/www/html/artisan octane:start --server=frankenphp --host=0.0.0.0 --port=8000 --admin-port=2019 --max-requests=1
...
```

**Reply by yilanboy** [reply]
This problem only occurs with octane, so I went back to using php fpm. 

I really not familiar with octane, I'm having some problems with xdebug on swoole.

**Reply by gazzoy** [reply]
Yes, seems like using Livewire(at least V3) w/ Octane could be a tricky one. Noting #7511 might be relate to this.

**Comment by gazzoy** [comment]
#8100 could be related to this as well.

Livewire might not be able to hydrate/dehydrate properly in subsequent requests in case we got ModelNotFoundException.

**Comment by gazzoy** [comment, marked as answer]
I dived into the code a bit, and here's what I found.

In normal case, Livewire w/ Octane processes like below. In this case, Livewire renders the page as expected.
 1) Laravel\Octane\Listeners\PrepareLivewireForNextOperation#handle
 2) Livewire\LivewireManager#flushState
 3) Livewire\Drawer\ImplicitRouteBinding (which resolves implicit model bindings in web.php)
 4) Livewire\Component#__invoke

On the other hand, in an empty page case, somehow Livewire only calls following functions. Which means Livewire never call `Livewire\Drawer\ImplicitRouteBinding` and `Livewire\Component#__invoke` so Livewire never resolves a model-bindings, and it doesn't render the page so we see an empty page.
 1) Laravel\Octane\Listeners\PrepareLivewireForNextOperation#handle
 2) Livewire\LivewireManager#flushState

 @joshhanley Do you happen to have any ideas in what cases Livewire never call both of `Livewire\Drawer\ImplicitRouteBinding` and `Livewire\Component#__invoke`?

**Reply by gazzoy** [reply]
@joshhanley 

Actually I could reproduce the issue and found out what the issue is! Created PR #8139 here, so please let me know if there's anything I can help you with.

**Reply by gazzoy** [reply]
@YilanBoy 

[PR](https://github.com/livewire/livewire/pull/8139) has been merged, so this issue should be fixed on the next release or so.

**Reply by yilanboy** [reply]
Really appreciate your help!
I have been following your PR and I have learned a lot from it, thank you!
